src/App.js

```diff
@@ -1,14 +1,142 @@
 import React from 'react';
 
+import Header from './Header';
+import Search from './Search';
+import Movies from './Movies';
+import Loading from './elements/Loading';
+import BackToTop from './elements/BackToTop';
+
+
 class App extends React.Component {
-  constructor(){
+  constructor() {
     super();
+    this.state = {
+      search: [],
+      searchTerm: "",
+      page: 2,
+      watchList: localStorage.getItem("watchList") ? localStorage.getItem("watchList") : [],
```

Comment:
You may need to perform JSON.parse on item retrieved from localStorage otherwise I expect it will just be a string